tools/AduCmdlets/create-adu-import-manifest.sh

```diff
@@ -30,7 +30,7 @@ write_verbose() {
         YELLOW="$(tput setaf 11)"
         RESET="$(tput sgr0)"
 
-        printf "${YELLOW}VERBOSE: $1${RESET}\n"
+        printf "%sVERBOSE: %s%s\n" "${YELLOW}" "$1" "${RESET}"
```

Comment:
>printf [](http://example.com/codeflow?start=8&length=6)

i could have used echo here.  Does it complain with echo? e.g.

`echo "${YELLOW}VERBOSE: $1${RESET}"`